docs/supported_platforms_and_configurations.md

```diff
@@ -37,9 +37,9 @@ For more information see [conan-io/conan-docker-tools](https://github.com/conan-
 ## Windows
 
 - Compilers: Visual Studio:
-  - 2015 (14.0.25431.01 Update 3)
-  - 2017 (15.9.19+28307.1000)
-  - 2019 (16.4.4+29728.190)
+  - 2015 (19.0.24215)
```

Comment:
seems weird?

Reply:
It is the version reported by `cl.exe`:

```
conan@C3I-STG-WIN1 C:\J\w\ValidateInfrastructure>"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\cl.exe" /?  1>nul 2>tmp.file 
[Pipeline] bat
[Pipeline] echo
Microsoft (R) C/C++ Optimizing Compiler Version 19.00.24215.1 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
```

Reply:
Are there several MSVC installations on the same host? That seems really unexcepted and more like a mismatch of different setups 🤔

Reply:
The agents have VS2015, VS2017 and VS2019 installed.